### PR TITLE
1.5 update, drawSize migration and new guns

### PR DESCRIPTION
**Pull request by SamaelGray**
## Additions

- What it says on the tin, also similar to: https://github.com/CombatExtended-Continued/CombatExtended/pull/3086
- Added the Boys ATR, Carl Gustaf and MosqueFAL.

## Changes

- Texture updates.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors

### CODE REVIEW

**Review by SerinaCE — COMMENTED** [review]
Would drop TP-82 and DoubleBadger too becaue useless, waiting for bossman on this

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Some remarks:
- As we discussed, split-off the Attachment merger stuff. It's its own discussion to be had, and having it in this PR is going going to slow down getting the necessary 1.5 update stuff. 
- For the LeMat:
  - It is a .44 caliber powder-and-ball revolver that was later converted to a pin-fire available in a 11 or 12mm cartridge. While I agree .44 magnum is more powerful side of things, the original gun is old enough we don't have great ammo options; we can either find an existing ammoset that's close and handwave that it was converted, or come up with a new one (mind, our current musket ball (slow) ammo is stat'd as .62 caliber).
  - Should probably increase the LeMat's reload time per round (and maybe make it load one at a time?)

**Comment by SamaelGray** [comment]
The Attachments merge is reverted. The main problem with the LeMat is its texture which is the old cap and ball variants instead of the cartridge one, that's the part that makes the .44 mag situation worse.

**Comment by Airomeda** [comment]
referencing from this PR on main mod https://github.com/CombatExtended-Continued/CombatExtended/pull/3135#issuecomment-2109894935, I think we need to add **CompProperties_Stunnable** to the shotgun turret.

**Comment by SamaelGray** [comment]
Should already be known https://github.com/CombatExtended-Continued/CombatExtended/issues/3024#issuecomment-2051335559

**Comment by SamaelGray** [comment]
> referencing from this PR on main mod [CombatExtended-Continued/CombatExtended#3135 (comment)](https://github.com/CombatExtended-Continued/CombatExtended/pull/3135#issuecomment-2109894935), I think we need to add `CompProperties_Stunnable` to the shotgun turret via the parent `TurretAutoBase` from main.

Should already be known https://github.com/CombatExtended-Continued/CombatExtended/issues/3024#issuecomment-2051335559

File: 1.5/Defs/ThingDefs/Apparel_CEA/HeavyCompositeHelmet.xml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+			<ArmorRating_Heat>0.54</ArmorRating_Heat>
+			<EquipDelay>1.5</EquipDelay>
+		</statBases>
+		<equippedStatOffsets>
+			<AimingAccuracy>-0.1</AimingAccuracy>
+			<Suppressability>-0.10</Suppressability>
+		</equippedStatOffsets>
+		<thingCategories Inherit='False'>
+			<li>ArmorHeadgear</li>
+		</thingCategories>
```
Comment (Airomeda) [inline 1.5/Defs/ThingDefs/Apparel_CEA/HeavyCompositeHelmet.xml:46]: redundant if ParentName is changed to `ArmorHelmetMakeableBase`

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**